src/libmongoc/tests/test-mongoc-loadbalanced.c

```diff
@@ -230,23 +230,16 @@ test_loadbalanced_client_uri_validation (void *unused)
    uri = mongoc_uri_new ("mongodb://localhost:27017");
    mongoc_uri_set_option_as_bool (uri, MONGOC_URI_LOADBALANCED, true);
    mongoc_uri_set_option_as_bool (uri, MONGOC_URI_DIRECTCONNECTION, true);
-   client = mongoc_client_new_from_uri (uri);
+   client = mongoc_client_new_from_uri_with_error (uri, &error);
 
-   ret = mongoc_client_command_simple (client,
-                                       "admin",
-                                       tmp_bson ("{'ping': 1}"),
-                                       NULL /* read prefs */,
-                                       NULL /* reply */,
-                                       &error);
+   BSON_ASSERT (!client);
    ASSERT_ERROR_CONTAINS (error,
-                          MONGOC_ERROR_SERVER_SELECTION,
-                          MONGOC_ERROR_SERVER_SELECTION_FAILURE,
+                          MONGOC_ERROR_COMMAND,
+                          MONGOC_ERROR_COMMAND_INVALID_ARG,
```

Comment:
This addresses a separate [test failure](https://spruce.mongodb.com/task/mongo_c_driver_asan_ubuntu_debug_compile_asan_clang_openssl_patch_a203bafab7d412ed05f33b99229bacbc043c2268_61cb7099850e6143937d4c52_21_12_28_20_16_28/logs?execution=0).